Shared/ContentView.swift

```diff
@@ -49,24 +39,46 @@ struct ContentView: View {
                 EmptyView()
             }
         }
+            
+        .fileImporter(isPresented: $presentFiles,
+                      allowedContentTypes: [.audio],
+                      allowsMultipleSelection: true,
+                      onCompletion: addSongHandler)
+    }
+    
+    func addSongHandler(result: Result<[URL], Error>?) {
+        guard let urls = try? result?.get() else { return print("Couldn't open files") }
+        
+        openSong (urls: urls)
         
     }
     
     func openSong (urls: [URL]) -> () {
         print("Reading URLS")
```

Comment:
[nit]

Is this really reading URLs anymore?

Reply:
I could probably remove this method entirely hand have that guard in the open song, thanks for the comment